d3a_api_client/redis_device.py

```diff
@@ -87,6 +85,10 @@ def select_aggregator(self, aggregator_uuid, is_blocking=True, unsubscribe_from_
                  f'{self._channel_prefix}/events/finish']
             )
 
+    def unselect_aggregator(self, aggregator_uuid):
+        """not needed ATM"""
```

Comment:
Would be better to raise this exception instead https://docs.python.org/3/library/exceptions.html#NotImplementedError with a descriptive message, in order for the user to be aware that this is not implemented/supported yet